msbuild/Directory.Build.props

```diff
@@ -1,6 +1,6 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 	<PropertyGroup>
-		<MessagingVersion>1.11.2</MessagingVersion>
+		<MessagingVersion>99.12.1</MessagingVersion>
```

Comment:
we jumped to 99????

Reply:
That's a testing version (we use 99.*.* as testing format). That's also why I added the do-not-merge tag, I just need the iOS packages with that test version IL merged so I can test